src/deltacode/cli.py

```diff
@@ -51,7 +51,12 @@ def write_json(deltacode, outfile, all_delta_types=False):
         ('delta_stats', deltacode.stats.to_dict()),
         ('deltas', deltas(deltacode, all_delta_types))
     ])
-
+    click.secho(get_notice(),fg="green")
+    click.echo('deltas_count: {}'.format(len([d for d in deltas(deltacode, all_delta_types)])))
+    click.echo('delta_stats: {')
+    for stat in deltacode.stats.to_dict():
+        click.echo("""   "{}": {}""".format(str(stat),deltacode.stats.to_dict()[stat]))
+    click.echo("}")    
```

Comment:
I do not think we need to struture the output like a json file. It adds a lot of noise. I would rather it just be output as `key: value` and not have JSON brackets or indentation

Reply:
@majurg sir , updated the content

Reply:
![Screenshot from 2020-03-14 00-49-12](https://user-images.githubusercontent.com/44219625/76653035-e3828f80-658d-11ea-8dc0-3782a45608da.png)